biosimulations/apps/simulators/src/app/simulators/view-simulator/view-simulator.component.html

```diff
@@ -47,21 +47,21 @@ <h2>
         </table>
       </div>
       <div class="column border">
-        <div class="section">
-          <h3>Run simulations online with runBioSimulations</h3>
+        <div class="section" *ngIf="simulator.image">
+          <h3>BioSimulators-compliant Docker image</h3>
           <div class="section-content">
             <ul class="icon-list">
               <li>
-                <biosimulations-icon icon="simulator"></biosimulations-icon>
-                <a [href]="dispatchAppRunUrl" target="_blank" class="text">
-                  {{ dispatchAppUrl }}
+                <biosimulations-icon icon="docker"></biosimulations-icon>
+                <a [href]="'https://github.com/orgs/biosimulators/packages/container/package/' + simulator.id" target="_blank" class="text">
```

Comment:
Will images necessarily be stored on our docker repo? I thought the idea was to allow authors to create their own images. Or is this intended to be a replicated version after submission?